wandb/sdk/artifacts/artifact.py

```diff
@@ -501,6 +480,57 @@ def metadata(self, metadata: dict) -> None:
         """
         self._metadata = self._normalize_metadata(metadata)
 
+    @property
+    def ttl(self) -> Union[datetime.timedelta, None]:
+        """Artifact TTL(time to live).
+
+        Set a TTL to mark the artifact for deletion after the given duration from Artifact creation has passed.
+        To turn off previously set artifact TTL, reset TTL to wandb.ArtifactTTL.DEFAULT.
+
+        Examples:
+            Basic usage
+            ```
+            artifact.ttl = timedelta(days=10) # artifact will be marked for deletion on artifact.created_at + 10 days
+            artifact.save() # or log the artifact if its new
+            ```
+        """
+        if self._ttl_change == ArtifactTTLChange.INHERITED:
```

Comment:
...then this becomes
```suggestion
        if self._ttl_changed and self._ttl_is_inherited:
```